tests/kernel/timer/timer_api/src/main.c

```diff
@@ -301,6 +428,10 @@ void test_timer_user_data(void)
 	}
 }
 
+/**
+ * @}
```

Comment:
close those on the top, and use `@ingroup kernel_timer_tests`

Reply:
I dont see this being updated.

Reply:
@nashif 
Sorry, i was not clear on the "ingroup" part. Updated now.
Thanks to @spoorthik for the clarification.